tauri-app/src/lib/components/tables/OrderTradesListTable.svelte

```diff
@@ -23,27 +23,31 @@
 
   $: orderTradesQuery = createInfiniteQuery({
     queryKey: [id, QKEY_ORDER_TRADES_LIST + id],
-    queryFn: ({ pageParam }: { pageParam: number }) => {
-      return orderTradesList(
+    queryFn: async ({ pageParam }: { pageParam: number }) => {
+      tradesCount = undefined;
+
+      const count = await orderTradesCount(id, $subgraphUrl || '', startTimestamp, endTimestamp);
```

Comment:
wouldn't it be better to do these in parallel rather than sequentially awaiting?

Reply:
Yeah it can be handled in the same promise, will implement it that way.